script.module.youtube.dl/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.youtube.dl" name="youtube-dl Control" version="21.204.0+matrix.1" provider-name="sy6sy2 based on ruuk's work">
```

Comment:
authors need to be separated by commas. Since this module is a patched version of youtube_dl, please use:
originalauthor, ruuk, you